src/main/java/com/shapesecurity/functional/data/ImmutableList.java

```diff
@@ -191,12 +190,36 @@
 
     @Override
     public final int hashCode() {
```

Comment:
The static analyzer cannot determine that `this.hashCode` is still nonnull at that point. Because indeed, this can be modified by multiple threads.